### ISSUE TITLE
NoMethodError: undefined method `oid'

### ISSUE THREAD

**Issue by Svashta** [plataformatec/devise#4769]
## Environment

- Ruby 2.4.0 (rvm)
- Rails 5.1.4
- Devise 4.4.1
- PostgreSQL 9.5 (ubuntu 16.04)

## Current behavior

When i want to run a default Devise migration (no changes to the file) I am getting:
NoMethodError: undefined method `oid' for  # ActiveRecord::ConnectionAdapters::ColumnDefinition:0x00000005817360>

## Expected behavior
Migration should run w/o problems.

Can please someone let me know if this is something with devise, pm gem, postgres version, rails version or some other configuration. I have no idea where to start solving this.
Some previous migrations from other gems are successfully migrated.

**Comment by Svashta** [comment]
Ok, so my users migration works normaly with sqlite3, also mysql, but not postgresql which i need. All other migrations work just fine on PG, which indicates there trully is something going on devise vise.

Please, any suggestions?

**Comment by tegon** [comment]
Hi @Svashta, thanks for your report.
I did a quick search and found some issues a Rails about it: https://github.com/rails/rails/issues/26137
Could you take a look to see if it's the same thing?

Thank you.

**Comment by tegon** [comment]
I'm closing this issue because it has not had recent activity.
If you're still facing this on the latest version, please open a new one with all the information requested in the template.

Thank you!

**Comment by asmrtfm** [comment]
New to rails.
I had what I believe you people refer to as "monkey patches" in config/initializers/array_extensions.rb that ended up being the culprit.

`strings()`, `to_strings()`, `ints()`, `to_ints()`

I don't have time to narrow down which but my money's on `strings` or `to_strings`

In any case, some combination of them are evidently used by PG gem and even though I wrapped them with `unless method_defined(...)` I suppose they are loaded before the PG gem.

I renamed them and my db:seed no longer fails with this error message.
